src/components/MainContent/MainContent.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useContext } from 'react';
+import {
+  ActivePostContext,
+  ActiveUserContext,
+  ErrorsContext,
+  LoaderContext,
+  PostsContext,
+} from '../../utils/Store';
+import { Loader } from '../Loader';
+import { PostsList } from './PostList/PostsList';
+import { ErrorTypes } from '../../types/ErrorTypes';
+
+export const MainContent: React.FC = () => {
+  const { activeUser } = useContext(ActiveUserContext);
+  const { activePost } = useContext(ActivePostContext);
+  const { posts } = useContext(PostsContext);
+  const { isError } = useContext(ErrorsContext);
+  const { isLoading } = useContext(LoaderContext);
```

Comment:
Consider renaming `isLoading` to `isLoad` to follow the checklist requirement: 'don't use `isLoad`, it can be `isLoading` or `isLoaded`'.